### DISCUSSION TITLE
Problem unmasked module esp8266 when in settings turn on "hide ssd #bug

### DISCUSSION THREAD

**Discussion by fghd3243223** [spacehuhn/esp8266_deauther#1637] [General]
after installing firmware 2.6 on esp8266 (nodemcu), when I choose in settings to hide ssd, earlier on a version below 2.1.0 on Windows it did not show the "hidden network"
now it shows like this. I don’t want to show "hidden networks" an inscription and unmasked the module esp8266 (nodemcu).
second problem: after the settings, the LED is turned off, after the esp8266 (nodemcu) module is turned on, the blue LED is on

**Comment by geo777535** [comment]
I Too see this big versions 2.6 .why such doing this version ?
